src/main/java/randoop/condition/RandoopConditionError.java

```diff
@@ -0,0 +1,33 @@
+package randoop.condition;
+
+/**
+ * Exception to represent errors that occur during creation or evaluation of a {@link Condition}.
```

Comment:
There were a couple throws that would be condition class configuration bugs, I changed these to BugInRandoopException.